ipn/ipnlocal/peerapi.go

```diff
@@ -778,6 +778,13 @@ func (h *peerAPIHandler) canPutFile() bool {
 // canDebug reports whether h can debug this node (goroutines, metrics,
 // magicsock internal state, etc).
 func (h *peerAPIHandler) canDebug() bool {
+	nm := h.ps.b.NetMap()
```

Comment:
add a comment about why we're not using selfNode

Reply:
Done.